docker-compose.yml

```diff
@@ -37,10 +38,30 @@ services:
       replicas: 3
     environment:
       <<: *common-variables
+  whatsticker-logger:
+    build: ./logger
+    restart: always
+    depends_on:
+      - redis
+      - metrics
+    command: bash -c "/wait && go run main.go"
+    environment:
+      <<: *common-variables
   redis:
     restart: always
     image: redis
-
+  metrics:
+    image: prom/pushgateway
+    command: ["--persistence.file","/tmp/whatsticker.metrics"]
+    restart: always
+    ports: 
+      - "9091:9091" 
+    expose:
+      - "9091"
+    volumes:
+      - type: bind
+        source: ./whatsticker.metrics
+        target: /tmp/whatsticker.metrics
```

Comment:
were you able to get the metrics to be bound to the persistence file?

Reply:
@derhnyel tell Onuh he has to add a `whatsticker.metrics` file to the repo in prod before redeploying

Reply:
Not yet , i can't `chown` on my current OS. I found a solution here [:](https://stackoverflow.com/questions/50009065/how-to-persist-data-in-prometheus-running-in-a-docker-container)

Reply:
Yeah.. I will .. Also i still need to make 1 more  commit for this to be merge ready